client/jobs.lua

```diff
@@ -1,95 +1,68 @@
-local currentLocation = 0
-currentBlip = nil
+local currentLocation = 1
+currentBlip = 0
 local isWorking = false
 
 -- Functions
 
-local function CreateJobBlip()
-    if currentLocation ~= 0 then
-        if DoesBlipExist(currentBlip) then
-            RemoveBlip(currentBlip)
-        end
-        currentBlip = AddBlipForCoord(Config.Locations.jobs[currentJob][currentLocation].coords.x, Config.Locations.jobs[currentJob][currentLocation].coords.y, Config.Locations.jobs[currentJob][currentLocation].coords.z)
-
-        SetBlipSprite (currentBlip, 402)
-        SetBlipDisplay(currentBlip, 4)
-        SetBlipScale  (currentBlip, 0.8)
-        SetBlipAsShortRange(currentBlip, true)
-        SetBlipColour(currentBlip, 1)
-
-        BeginTextCommandSetBlipName("STRING")
-        AddTextComponentSubstringPlayerName(Config.Jobs[currentJob])
-        EndTextCommandSetBlipName(currentBlip)
+function CreateJobBlip(first) -- Used globally
+    if DoesBlipExist(currentBlip) then
+        RemoveBlip(currentBlip)
+    end
+    local coords = Config.Locations.jobs[currentJob][currentLocation].coords.xyz
+    currentBlip = AddBlipForCoord(coords.x, coords.y, coords.z)
+    SetBlipSprite(currentBlip, 402)
+    SetBlipDisplay(currentBlip, 4)
+    SetBlipScale(currentBlip, 0.8)
+    SetBlipAsShortRange(currentBlip, true)
+    SetBlipColour(currentBlip, 1)
+    BeginTextCommandSetBlipName("STRING")
+    AddTextComponentSubstringPlayerName("Prison Work")
+    EndTextCommandSetBlipName(currentBlip)
+    if first then return end
+    local Chance = math.random(100)
+    local Odd = math.random(100)
+    if Chance ~= Odd then return end
+    TriggerServerEvent('QBCore:Server:AddItem', 'phone', 1)
+    TriggerEvent('inventory:client:ItemBox', QBCore.Shared.Items["phone"], "add")
+    QBCore.Functions.Notify(Lang:t("success.found_phone"), "success")
+end
 
-        local Chance = math.random(100)
-        local Odd = math.random(100)
-        if Chance == Odd then
-            TriggerServerEvent('QBCore:Server:AddItem', 'phone', 1)
-            TriggerEvent('inventory:client:ItemBox', QBCore.Shared.Items["phone"], "add")
-            QBCore.Functions.Notify(Lang:t("success.found_phone"), "success")
+local function CheckAllLocations()
+    local amount = 0
+    for i = 1, #Config.Locations.jobs["electrician"] do
+        local current = Config.Locations.jobs["electrician"][i]
```

Comment:
Is there any way to not hardcode the `electrician` key ? I can do it if you want it :]

Reply:
As I said earlier, I didn't have much time to make this, this was one of the last things I did so I just went hardcoded because everything else pretty much is already, so I can make it universal tomorrow